buildSrc/src/main/groovy/org/apache/beam/gradle/BeamModulePlugin.groovy

```diff
@@ -494,7 +494,7 @@ class BeamModulePlugin implements Plugin<Project> {
     def sbe_tool_version = "1.25.1"
     def slf4j_version = "1.7.30"
     def spark2_version = "2.4.8"
-    def spark3_version = "3.1.2"
+    def spark3_version = "3.2.0"
```

Comment:
Why not 3.2.2?

Reply:
Btw 3.2.0 / 3.2.1 contain CVEs

Reply:
Wow, tests are failing... looks like 3.2.0 to 3.2.1 (which is currently tested as part of `sparkVersionsTest`) contains a breaking change